src/ListviewSort.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
+<widget id="com.mendix.widget.custom.ListviewSort.ListviewSort" offlineCapable="true" needsEntityContext="false" xmlns="http://www.mendix.com/widget/1.0/">
```

Comment:
`com.mendix.widget.custom.listviewsort.ListviewSort` only last class name in a reverse domain name should be CamelCased

Reply:
done